pkgs/top-level/all-packages.nix

```diff
@@ -9495,8 +9495,6 @@ with pkgs;
 
   keyfuzz = callPackage ../tools/inputmethods/keyfuzz { };
 
-  keymapp = callPackage ../applications/misc/keymapp { };
-
```

Comment:
Is this expected? I thought you need this entry in order to reference the package by `nixkpgs#keymapp`.

Reply:
`pkgs/by-name` are not exposed in that File anylonger unless one needs overrides.
So I removed it from the top-level s manually exposure
See https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#name-based-package-directories for further ref.